intel_extension_for_transformers/neural_chat/pipeline/plugins/audio/tts.py

```diff
@@ -61,11 +61,11 @@ def __init__(self, output_audio_path="./response.wav", voice="default", stream_m
                           + '/assets/speaker_embeddings/spk_embed_default.pt'):
             default_speaker_embedding_path = os.path.split(os.path.split(os.path.split(script_dir)[0])[0])[0] \
                               + '/assets/speaker_embeddings/spk_embed_default.pt'
-        elif os.path.exists(os.path.join(asset_path, 'speaker_embeddings/spk_embed_default.pt')):
+        elif os.path.exists(os.path.join(asset_path, 'speaker_embeddings/spk_embed_default.pt')): # pragma: no cover
```

Comment:
why checking the same file multiple times from different places?

typically, if you want to bypass the code coverage checking, you should use assert instead of if/else.

Reply:
Will check that with yuxiang.